src/ChirpWeb/Pages/CreatingCheep.cshtml

```diff
@@ -0,0 +1,29 @@
+@page "/CreateCheep"
+@model ChirpWeb.Pages.CreateCheepModel
+@{
+    ViewData["Title"] = "Create Cheep";
+    Layout = "_Layout";
+}
+
+<h2>Create a New Cheep</h2>
+
+<form method="post">
+    @Html.AntiForgeryToken()
```

Comment:
This is a security feature to help protect your application against cross-site request forgery.

Source: https://stackoverflow.com/questions/44783702/what-is-the-use-of-html-antiforgerytoken